plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -386,118 +387,120 @@ decodeBuiltin = dBEBits8 builtinTagWidth
-              LessThanEqualsInteger -> 6
-              EqualsInteger         -> 9
-              AppendByteString      -> 10
-              TakeByteString        -> 11
-              DropByteString        -> 12
-              Sha2_256              -> 13
-              Sha3_256              -> 14
-              VerifySignature       -> 15
-              EqualsByteString      -> 16
-              QuotientInteger       -> 17
-              ModInteger            -> 18
-              LessThanByteString    -> 19
-              GreaterThanByteString -> 20
-              IfThenElse            -> 21
-              Append                -> 23
-              Trace                 -> 24
-              -- 25 unused
-              -- 26 unused
-              -- 27 unused
-              EqualsString          -> 28
-              EncodeUtf8            -> 29
-              DecodeUtf8            -> 30
-              FstPair               -> 31
-              SndPair               -> 32
-              NullList              -> 33
-              HeadList              -> 34
-              TailList              -> 35
-              ConstrData            -> 36
-              MapData               -> 37
-              ListData              -> 38
-              IData                 -> 39
-              BData                 -> 40
-              UnConstrData          -> 41
-              UnMapData             -> 42
-              UnListData            -> 43
-              UnIData               -> 44
-              UnBData               -> 45
-              EqualsData            -> 46
-              ChooseData            -> 47
-              ChooseUnit            -> 48
-              MkPairData            -> 49
-              MkNilData             -> 50
-              MkNilPairData         -> 51
-              MkCons                -> 52
-              ChooseList            -> 53
-              Blake2b_256           -> 54
-              LengthOfByteString    -> 55
-              IndexByteString       -> 56
-              ConsByteString        -> 57
-              SliceByteString       -> 58
+              AddInteger               -> 0
+              SubtractInteger          -> 1
+              MultiplyInteger          -> 2
+              DivideInteger            -> 3
+              QuotientInteger          -> 4
+              RemainderInteger         -> 5
+              ModInteger               -> 6
+              EqualsInteger            -> 7
+              LessThanInteger          -> 8
+              LessThanEqualsInteger    -> 9
+
+              AppendByteString         -> 10
+              ConsByteString           -> 11
+              SliceByteString          -> 12
+              LengthOfByteString       -> 13
+              IndexByteString          -> 14
+              EqualsByteString         -> 15
+              LessThanByteString       -> 16
+              LessThanEqualsByteString -> 17
+
+              Sha2_256                 -> 18
```

Comment:
I think we're expecting to add more cryptographic things quite soon.  Will we put them at the end, or what? I suppose there's no point leaving a gap because we don't know how big to make it.

Reply:
Yeah, I think we just have to accept that additional things go in a disorganized bundle at the end.